website/src/home/content/about.md

```diff
@@ -0,0 +1,15 @@
+# About
+
+Compa is a companion application for students to manage and access resources at their higher education institution.
```

Comment:
Let's use:

Compa is an open initiative to promote collaboration and knowledge sharing among students of higher education.